src/App.vue

```diff
@@ -1,78 +1,21 @@
 <!-- Use preprocessors via the lang attribute! e.g. <template lang="pug"> -->
 <template>
   <div id="app">
-    <b-input-group class="p-5" prepend="New Todo">
-      <b-form-input v-model="newTodo"></b-form-input>
-      <b-input-group-append>
-        <b-button v-on:click="addTodo">Add</b-button>
-      </b-input-group-append>
-    </b-input-group>
+    <h1>Todo List</h1>
 
-    <ul class="list-group p-5">
-      <li
-        v-for="(item, index) in todos"
-        :key="index"
-        class="
-          list-group-item
-          d-flex
-          justify-content-between
-          align-items-center
-        "
-      >
-        <label v-if="!item.done">{{ item.title }}</label>
-        <del v-else>{{ item.title }}</del>
+    <p>
+      <router-link to="/">
+        <b-button>Home</b-button>
```

Comment:
BootstrapVue Button supports router-link with "to".
[https://bootstrap-vue.org/docs/components/button#router-link-support](url)